src/main/java/com/amazon/kinesis/streaming/agent/custom.log4j.xml

```diff
@@ -10,9 +10,7 @@
         <param name="MaxBackupIndex" value="5" />
         <param name="MaxFileSize" value="10485760" />
         <param name="Append" value="true" />
-        <layout class="org.apache.log4j.PatternLayout">
-            <param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss.SSSZ} %X{hostname} (%t) %c [%p] %m%n" />
-        </layout>
+        <layout class="net.logstash.log4j.JSONEventLayoutV1" />
```

Comment:
Shouldn't this be there in FALLBACK appender too in case we want a fallback option for lgos?.

Reply:
The fallback part I didn't touch, I think that part is there in case there are issues logging with the main FILE appender